src/comments.ts

```diff
@@ -58,15 +58,15 @@ ${names.map(n => `${n}`).join(" ")}
 </details>`
 });
 
-export const PingStaleReviewer = (reviewedAbbrOid: string, reviewers: string[]) => ({
-    tag: `stale-ping-${sha256(reviewers.join("-")).substr(0, 6)}-${reviewedAbbrOid}`,
+export const PingStaleReviewer = (reviewedOid: string, reviewers: string[]) => ({
+    tag: `stale-ping-${sha256(reviewers.join("-")).substr(0, 6)}-${reviewedOid.slice(0, 7)}`,
     status: `@${reviewers.join(", @")} Thank you for reviewing this PR! The author has pushed new commits since your last review. Could you take another look and submit a fresh review?`
 });
 
 export const OfferSelfMerge = deletedWhenNotPresent("merge-offer", tag =>
-    (user: string, otherOwners: string[], abbrOid: string) => ({
-        // Note: pr-info.ts searches for the `(at ${abbrOid})`
-        tag, status: `@${user} Everything looks good here. Great job! I am ready to merge this PR (at ${abbrOid}) on your behalf.
+    (user: string, otherOwners: string[], headOid: string) => ({
+        // Note: pr-info.ts searches for the `(at ${headOid})`
+        tag, status: `@${user} Everything looks good here. Great job! I am ready to merge this PR (at ${headOid}) on your behalf.
```

Comment:
See my reply to the other comment: I much prefer if this is kept in its abbreviated form.

Reply:
Done :heavy_check_mark: